### ISSUE TITLE
v 91.53 can not home or move z in printer mode directly after homing z in milling mode

### ISSUE THREAD

**Issue by drillabit** [RF1000/Repetier-Firmware#9]
Homing z in milling mode leaves lower z switch pressed. When switching to printer mode, any attempt to move z upwards fails because of the pressed switch. Homing will then drive the z axis to a crash with the lower z-switch fixture (the screw next to the switch) and fail.

**Comment by Cirrus01** [comment]
Agree, I had this after Object Output and then switching to milling mode. You should asign this to RF1000.

**Comment by RF1000** [comment]
Hello,

we are going to improve the z-endstop checks in the next version of the firmware in order to avoid this trap.
